sycl/doc/SYCLCompilerUserManual.md

```diff
@@ -81,6 +81,27 @@ your application.
     specified with `-foffload-static-lib` are treated as host libraries and are
     only used during the final host link.
 
+**`-fsycl-device-code-split=<value>`**
+
+    Perform SYCL device code split. There are three possible values for this
+    option:
+    - per_kernel - a separate device code module is created for each SYCL
+    kernel. Each device code module will contain a kernel and all its
+    dependencies, i.e. called functions and used variables.
+    - per_source - a separate device code module is created for each source
+    (translation unit). Each device code module will contain a bunch of kernels
+    grouped on per-source basis and all their dependencies, i.e. all used
```

Comment:
```suggestion
    (grouped on per-source basis) and all their dependencies, i.e. all used
```

Reply:
I'd like to leave this phrase without braces because it's kind of key information